app/pages/shows/_slug/index.vue

```diff
@@ -24,7 +24,7 @@
         <div>{{ arcsiInfosBlock.description }}</div>
       </div>
     </div>
-    <div>
+    <div v-if="arcsiShowsList && arcsiShowsList.length">
```

Comment:
I'm never sure how js handles certain truthiness situations but what happens here if `arcsiShowsList` is not available at all? 

In that case wouldn't `arcsiShowsList.length` throw a runtime error because `undefined has no property called 'length'` or whatever the message is?